tests/expected/function-contract/modifies/havoc_pass.expected

```diff
@@ -0,0 +1,38 @@
+copy_replace.assertion\
+- Status: SUCCESS\
+- Description: "equality"\
+in function copy_replace
+
+VERIFICATION:- SUCCESSFUL
+
+copy.assigns\
+- Status: SUCCESS\
+- Description: "Check that var_4 is assignable"\
```

Comment:
Checking the MIR variable might be brittle.

Reply:
Sadly, as  mentioned in the description of the PR, there's not really a good way to specify these checks in `.expected` files. If you have suggestions how to work around this I'm all ears.

Reply:
could you at least use a regular expression here like `var_*`?

Reply:
I checked and the test runner does not (currently) support regular expressions. I could add this capability but that would perhaps need to be a separate PR. Would you like me to try?

Reply:
Nope. We can track this in a github issue and merge it on a separate PR. Not a blocker for this one.